pypika/tests/test_groupby_modifiers.py

```diff
@@ -142,3 +142,14 @@ def test_verticaoracle_multiple_rollups(self):
         )
 
         self.assertEqual('SELECT "foo","fiz",SUM("bar") FROM "abc" GROUP BY ROLLUP("foo"),ROLLUP("fiz")', str(q))
+
+
+    def test_verticaoracle_groupby_with_alias(self):
```

Comment:
This doesn't belong in the RollupTests class. Can you please move it to `GroupByTests` in `test_selects`?